composer.json

```diff
@@ -24,32 +24,32 @@
         }
     ],
     "require": {
-        "php": "^7.2 || ^8.0",
+        "php": "^7.3 || ^8.0",
```

Comment:
Shall we drop 7.3 as well?

Reply:
I don't see a reason to, but also, I don't see a reason not to, so sure, let's do it!